File: Beecrowd/1099 Soma de Ímpares Consecutivos II/main.cpp
#include <iostream>

using namespace std;

int main()
{
    int n, x, y, maior, menor, soma;

    cin >> n;

    for (int i = 0; i < n; i++)
    {
        cin >> x >> y;

        menor = x;
        maior = y;

        if (x > y)
        {
            menor = y;
            maior = x;
        }

        soma = 0;
        for (int j = menor + 1; j < maior; j++)
        {
            if (j % 2 != 0)
                soma += j;
        }
        cout << soma << endl;
    }

    return 0;
}

File: Beecrowd/1099 Soma de Ímpares Consecutivos II/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        int n, x, y, maior, menor;

        n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            x = sc.nextInt();
            y = sc.nextInt();

            menor = x;
            maior = y;

            if (x > y) {
                menor = y;
                maior = x;
            }

            int soma = 0;
            for (int j = menor + 1; j < maior; j++) {
                if (j % 2 != 0)
                    soma += j;
            }
            System.out.println(soma);
        }

        sc.close();
    }
}